README.md

```diff
@@ -106,37 +106,35 @@ fail-on-scopes:
 
 ### allow-licenses
 
-Only allow the licenses in this list. See "[Licenses](https://github.com/actions/dependency-review-action#licenses)".
+Only allow the licenses that comply with the expressions in this list. See "[Licenses](https://github.com/actions/dependency-review-action#licenses)".
 
-**Possible values**: Any `spdx_id` value(s) from
-https://docs.github.com/en/rest/licenses.
+**Possible values**: Any valid [spdx license expression](https://spdx.dev/spdx-specification-21-web-version/#h.jxpfx0ykyb60).
```

Comment:
```suggestion
**Possible values**: A list of of [SPDX-compliant license identifiers](https://spdx.org/licenses/).
```

This URL contains a list of the allowed expressions by listing them instead of forcing users to read the entire spec 😅